### DISCUSSION TITLE
Only push on one machine

### DISCUSSION THREAD

**Discussion by finn-matti** [Vinzent03/obsidian-git#206] [Q&A]
I would like to use Obsidian Git as an additional backup of my notes. However I can't figure out how to only make it push on one of my two machines. Is this currently possible?

**Comment by Vinzent03** [comment]
You can either do backups manually on only one machine, or you add the plugin's config to `.gitignore` and specify two different configurations. Enable auto backup only on one machine.

**Reply by finn-matti** [reply]
Your second idea won't work if I'm using another solution to sync (e. g. Obsidian Sync), because the plugin config is synced in any case. So there is only manual backups at the moment?

**Reply by Vinzent03** [reply]
I don't know Obsidian sync very well, but cannot you turn sync off for specific plugins? If not, you are right, there is currently no way to fix this.

**Reply by finn-matti** [reply]
It would be nice, if you could specify this. It could be done using the hostname variable, I would think. So I guess this has now become a feature request. :)

**Reply by finn-matti** [reply]
See: #207
